content/ember-released-5-3.md

```diff
@@ -0,0 +1,128 @@
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 5.3
+
+Ember.js 5.3 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Deprecations
+
+Ember.js 5.3 introduced 1 deprecation.
+
+1. Deprecate implicit record loading in Ember Route per [RFC #774](https://rfcs.emberjs.com/id/0774-implicit-record-route-loading). Please see [the deprecation guide](https://deprecations.emberjs.com/v5.x#toc_deprecate-implicit-route-model) for more information.
+<!-- Block start: If there were no deprecations, remove this block -->
+
+Deprecations are added to Ember.js when an API will be removed at a later date. Each deprecation has an entry in the deprecation guide describing the migration path to a more stable API. Deprecated public APIs are not removed until a major release of the framework.
+
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+<!-- Block end -->
+
+For more details on changes in Ember.js 5.3, please review the [Ember.js 5.3.0 release page](https://github.com/emberjs/ember.js/releases/tag/v5.3.0).
+
+---
+
+## EmberData
+
+EmberData is the official data persistence library for Ember.js applications.
+
+### Changes in EmberData 5.3
+
+#### Bug Fixes
+
+EmberData 5.3 introduced 10 bug fixes. Details are in the
+[EmberData 5.3.0 release page](https://github.com/emberjs/data/releases/tag/v5.3.0).
+
+#### Features
+
+EmberData 5.3 introduced several new features.
+
+This release primarily focused on polish for the `RequestManager` story, introducing builder utils, builders for `REST` `ActiveRecord` and `JSON:API` requests as well as serialization utils for `JSON:API` requests.
+
+As part of shipping serialization utils, we implemented several long-awaited cache features to support them:
+
+- `cache.changedRelationships` retrieves a diff of what has been added or removed from each relationship for a resource
+- `cache.hasChangedRelationships` answers whether any relationships have changed for a resource
+- `cache.rollbackRelationships` will reset all relationships for that resource to their "remote" state, removing any local changes.
+
+#### Deprecations
+
+EmberData 5.3 introduced 5 deprecations.
```

Comment:
Can we get this merged? https://github.com/ember-learn/deprecation-app/pull/1339

Reply:
@kategengler Yes, I'll get it done today.

Reply:
✅